editor/debugger/script_editor_debugger.cpp

```diff
@@ -1192,6 +1192,19 @@ void ScriptEditorDebugger::_method_changed(Object *p_base, const StringName &p_n
 	}
 
 	if (node) {
+		// Do not send the root transform of edited scene.
```

Comment:
I would expand on this comment and make it clear why we're doing this. I bet someone is going to raise a bug report at some point because they right clicked on a node, saved branch to subscene, and then expect manipulating the root node when editing the subscene does something. As mentioned before, this is the exception to the rule where we could expect this to do something.

In all other scenarios, especially when scenes are instanced by code or when multiple instances are present, the root node of a subscene has no meaning, the transform in the parent scene will always overrule it.

One improvement we could do is to ensure the inspector on the root node is disabled in this condition, but my knowledge of the editor is too limited to suggest how to put this into practice.

Reply:
What's the best practice when it comes to mentioning additional context like this that might need a lot of text? Should I just insert it all inline, or can we actually just include pull request numbers?

Reply:
I have expanded the comments.